From 4063e89423325632efc734ae3d2e143f354392cf Mon Sep 17 00:00:00 2001
From: Ricardo Wurmus <rekado@elephly.net>
Date: Fri, 18 Dec 2020 10:35:39 +0100
Subject: [PATCH 3/7] NewProjectWizard: Unhide version selector widget.

*
src/gwt/src/org/rstudio/studio/client/projects/ui/newproject/NewProjectWizard.java:
Do not hide the widget.
---
 .../studio/client/projects/ui/newproject/NewProjectWizard.java   | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/gwt/src/org/rstudio/studio/client/projects/ui/newproject/NewProjectWizard.java b/src/gwt/src/org/rstudio/studio/client/projects/ui/newproject/NewProjectWizard.java
index f386b99af1..d39ef19bf9 100644
--- a/src/gwt/src/org/rstudio/studio/client/projects/ui/newproject/NewProjectWizard.java
+++ b/src/gwt/src/org/rstudio/studio/client/projects/ui/newproject/NewProjectWizard.java
@@ -71,7 +71,6 @@ public class NewProjectWizard extends Wizard<NewProjectInput,NewProjectResult>
          rVersionSelector_.setRVersion(rVersion);
          addLeftWidget(rVersionSelector_);
          rVersionSelector_.getElement().getStyle().setMarginRight(8, Unit.PX);
-         rVersionSelector_.setVisible(false);
       }
       
       openInNewWindow_ = new CheckBox("Open in new session");
-- 
2.33.0

